assignments/this.js

```diff
@@ -13,14 +13,49 @@
 
 // code example for Window Binding
 
+function vampire() {
```

Comment:
My last comments were incorrect. Just try `console.log(this)` and notice how it gives you the entire contents of the window object (your browser).